tests/utils.go

```diff
@@ -465,6 +465,8 @@ func WaitForVirtualMachineInstancePhase(client kubecli.KubevirtClient, namespace
 }
 
 func WaitForVirtualMachineStatus(client kubecli.KubevirtClient, namespace, name string, statuses ...kvv1.VirtualMachinePrintableStatus) error {
+	fmt.Fprintf(ginkgo.GinkgoWriter, "Waiting for any of %s statuses\n", statuses)
```

Comment:
You are using `By(fmt.Sprintf("..."))` in other places, why not here?

Reply:
updated